app/assets/stylesheets/pagination.css.scss

```diff
@@ -0,0 +1,13 @@
+.pagination a, .pagination span.current, .pagination span.gap {
```

Comment:
nice try, but use this https://github.com/matenia/bootstrap-kaminari-views

the problem in your code with float: left. it's ALWAYS required to clear"float" elements. In your case just remove this CSS and use gem above. It will use native bootstrap styles